schema.graphql

```diff
@@ -171,11 +171,15 @@ type ProposedArtistAddressesAndSplit @entity {
   createdAt: BigInt!
 }
 
-type EngineRegistry @entity {
-  "Unique identifier made up of the Engine Registry's contract address"
+type CoreRegistry @entity {
+  """
+  Unique identifier made up of the Core Registry's contract address (legacy name was Engine Registry).
+  Note: For legacy MinterFilters, this is a dummy ID, equal to the address of the single core contract
+  associated with the minter filter.
+  """
   id: ID!
 
-  "Core contracts that are registered on this Engine Registry, when this is most recent Engine Registry to add the contract"
+  "All core contracts that are registered on this CoreRegistry, when this is most recent Core Registry to add the contract"
```

Comment:
Just curious, do we ever expect a contract would be registered on more than one registry?

Reply:
I don't think so, at least on in a meaningful way where we would want to know anything but the most recent core registry.
I'm going to resolve this, but @jakerockland feel free to chime in if you have any additional thoughts about this.